content/src/licenses/certified-public-accountant.md

```diff
@@ -15,6 +16,12 @@ callToActionText: Renew My Registration
 - Current information for the Resident Manager-in-Charge, this must be the Certified Public Accountant in the firm
 - Pay the associated fees
 
-> **Once completed, you will have:**
->
-> - A renewed accounting firm registration valid for three years. It will be mailed to your business mailing address on file
+:::callout{ icon="false" calloutType="conditional" }
+
+- A renewed accounting firm registration valid for three years. It will be mailed to your business mailing address on file
+
+:::
+
+:::callout{ header="What if I don't renew my registration?" icon="false" calloutType="conditional" }
+If you don't renew your registration 30 days after expiration, your license will be suspended and you won't be able to legally run your business. You need to contact your licensing board for reinstatement procedures and associated fees.
+:::
```

Comment:
[boulder]

```suggestion
:::callout{ header="What if I don't renew my registration?" icon="false" calloutType="conditional" }

- A renewed accounting firm registration valid for three years. It will be mailed to your business mailing address on file

 If you don't renew your registration 30 days after expiration, your license will be suspended and you won't be able to legally run your business. You need to contact your licensing board for reinstatement procedures and associated fees.
:::
```